dags/annotate.py

```diff
@@ -18,30 +19,38 @@
     intro = BashOperator(task_id='Intro',
                          bash_command='echo running tranql translator && exit 0',
                          dag=dag)
-    # make_kg_tagged = create_python_task(dag, "create_kgx_files", DugUtil.make_kg_tagged)
 
     # Unzip and get files, avoid this because
     # 1. it takes a bit of time making the dag itself, webserver hangs
-    # 2. Every task in this dag would still need to execute this part
-    # making it redundant
+    # 2. Every task in this dag would still need to execute this part making it redundant
     # 3. tasks like intro would fail because they don't have the data dir mounted.
 
-    get_topmed_files = create_python_task(dag, "get_topmed_data", get_topmed_files)
-    extract_db_gap_files = create_python_task(dag, "get_dbgap_data", get_dbgap_files)
-
-    annotate_topmed_files = create_python_task(dag, "annotate_topmed_files", DugUtil.annotate_topmed_files)
-    annotate_db_gap_files = create_python_task(dag, "annotate_db_gap_files", DugUtil.annotate_db_gap_files)
-
     make_kg_tagged = create_python_task(dag, "make_tagged_kgx", DugUtil.make_kg_tagged)
 
     dummy_stepover = DummyOperator(
         task_id="continue",
     )
 
     #intro >> run_printlog
-    intro >> get_topmed_files >> annotate_topmed_files >> dummy_stepover
-    intro >> extract_db_gap_files >> annotate_db_gap_files >> dummy_stepover
-    dummy_stepover >> make_kg_tagged
+    envspec = os.getenv("ROGER_DUG__INPUTS_DATA__SETS","topmed")
+    data_sets = envspec.split(",")
+
+    clear_annotation_items = create_python_task(dag, "clear_annotation_files", DugUtil.clear_annotation_cached)
+
+    for i, data_set in enumerate(data_sets):
```

Comment:
@YaphetKG are we all on board with, 
1) making this a dynamic task  generator, and 
2) hard-coding each step instead of making it metadata-driven?